examples/appsync/function.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: appsync.aws.upbound.io/v1beta1
+kind: Function
+metadata:
+  annotations:
+    meta.upbound.io/example-id: appsync/v1beta1/function
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    apiIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    dataSourceSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    name: example
+    region: us-west-1
+    requestMappingTemplate: |
+      {
+          "version": "2018-05-29",
+          "method": "GET",
+          "resourcePath": "/",
+          "params":{
+              "headers": $utils.http.copyheaders($ctx.request.headers)
+          }
+      }
+    responseMappingTemplate: |
+      #if($ctx.result.statusCode == 200)
+          $ctx.result.body
+      #else
+          $utils.appendError($ctx.result.body, $ctx.result.statusCode)
```

Comment:
Do we need a manual intervention here?

Reply:
No need for manual intervention, this is handled by provider-aws.